aws/cloudformation/application.yaml

```
@@ -0,0 +1,332 @@
+      ContainerDefinitions:
+        - Name: backend
+          Image: !Ref BackendImageUri
+          Essential: true
+          PortMappings:
+            - ContainerPort: 5000
+              Protocol: tcp
+          Environment:
+            - Name: FLASK_ENV
+              Value: !Ref Environment
+            - Name: DATABASE_URL
+              Value: !Sub 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)